### PR TITLE
Add `--json-value` flag

### LINKED ISSUE #16: is it possible to save array values?
I'm trying to save the following value to my package.json file

```json
  "jest": {
    "moduleFileExtensions": [
      "js",
      "ts",
      "svelte"
    ]
  }
```

### PR DESCRIPTION
**Pull request by fregante**
Close #16

### CODE REVIEW

File: lib/parse-value.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		try {
+			return JSON.parse(value)
+		} catch (error) {
+			throw new Error(
+				util.format(
+					'The received value cannot be parsed as a JSON string. Executed:\n'
+					+ 'JSON.parse(%j).\n'
+					+ 'You might want to wrap the string in single quotes, for example:\n'
+					+ 'dot-json file.json key \'{"value": 42}\' --json-value',
+					value
```
Comment (fregante) [inline lib/parse-value.js:14]: The native message by `JSON.parse` is pretty useless and that's compounded by the string handling by the console. I thought it'd be best to show exactly what's being executed and how to fix it

Screenshot

**Comment by fregante** [comment]
Happy Easter ping ☺️ 🕊️

**Comment by maikelvl** [comment]
Thanks for the MR. 

@fregante Can you add the new option to the README as well?

**Comment by fregante** [comment]
Wow, time flies! 

Done

**Pull request merged**